pkg/cmd/cli/describe/describer.go

```diff
@@ -377,7 +377,11 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 	if err != nil {
 		return "", err
 	}
-	buildList, err := d.Builds(namespace).List(labels.SelectorFromSet(labels.Set{buildapi.DeprecatedBuildConfigLabel: name}), fields.Everything())
+	buildConfigLables := labels.Set{
+		buildapi.BuildConfigLabel: name,
```

Comment:
@bparees if you add both labels in a selector then you'll handle only builds that have both labels... This is the issue upstream about enchancing selectors: https://github.com/kubernetes/kubernetes/issues/15494

@liggitt how about having one `labels.Everything` query and filter locally twice instead?